example/lib/trip_calculation_example.dart

```diff
@@ -0,0 +1,215 @@
+import 'dart:async';
+import 'dart:math';
+
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:maplibre_gl/mapbox_gl.dart';
+import 'package:maplibre_gl_example/page.dart';
+
+class TripCalculationPage extends ExamplePage {
+  const TripCalculationPage({super.key})
+      : super(const Icon(Icons.trip_origin), 'Trip calculation example');
```

Comment:
I like the idea, but could we make it clear that this plugin does not actually support "real" route calculations, e.g. with a comment or similar? Otherwise people may have unrealistic hopes 😄

Reply:
Yes, I also thought about renaming this. 👍🏽